### PR TITLE
updated domo docs

### PR DESCRIPTION
**Pull request by NiharDoshi99**
### Describe your changes :

I worked on updating docs for domo on `scopes`

#
### Type of change :

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

### CODE REVIEW

File: openmetadata-docs/content/connectors/dashboard/domo-dashboard/airflow.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ To deploy OpenMetadata, check the <a href="/deployment">Deployment</a> guides.
 To run the Ingestion via the UI you'll need to use the OpenMetadata Ingestion Container, which comes shipped with
 custom Airflow plugins to handle the workflow deployment.
 
+<Note>
+
+For metadata ingestion, kindly make sure add alteast `dashboard` scopes to the clientId provided.
+Question related to scopes, click [here](https://developer.domo.com/docs/authentication/quickstart-5)
```
Comment (nahuelverdugo) [inline openmetadata-docs/content/connectors/dashboard/domo-dashboard/airflow.md:26]:
```suggestion
Question related to scopes, click [here](https://developer.domo.com/docs/authentication/quickstart-5).
```

File: openmetadata-docs/content/connectors/pipeline/domo-pipeline/index.md [hunk 2]
```diff
@@ -40,6 +40,13 @@ To deploy OpenMetadata, check the <a href="/deployment">Deployment</a> guides.
 To run the Ingestion via the UI you'll need to use the OpenMetadata Ingestion Container, which comes shipped with
 custom Airflow plugins to handle the workflow deployment.
 
+<Note>
+
+For metadata ingestion, kindly make sure add alteast `data` scopes to the clientId provided.
```
Comment (nahuelverdugo) [inline openmetadata-docs/content/connectors/pipeline/domo-pipeline/index.md:45]: Do pipelines and databases share the same scope?

**Review by OnkarVO7 — DISMISSED** [review]

File: openmetadata-docs/content/connectors/dashboard/domo-dashboard/airflow.md (hunk 1 above)
Comment (nahuelverdugo) [reply openmetadata-docs/content/connectors/dashboard/domo-dashboard/airflow.md:26]: nit: all those links do not finish with a dot at the end. We can be more consistent with the format of the text.

File: openmetadata-docs/content/connectors/dashboard/domo-dashboard/cli.md [hunk 3]
```diff
@@ -20,6 +20,13 @@ To deploy OpenMetadata, check the <a href="/deployment">Deployment</a> guides.
 To run the Ingestion via the UI you'll need to use the OpenMetadata Ingestion Container, which comes shipped with
 custom Airflow plugins to handle the workflow deployment.
 
+<Note>
+
+For metadata ingestion, kindly make sure add alteast `dashboard` scopes to the clientId provided.
+Question related to scopes, click [here](https://developer.domo.com/docs/authentication/quickstart-5)
```
Comment (nahuelverdugo) [inline openmetadata-docs/content/connectors/dashboard/domo-dashboard/cli.md:26]: Why do some links point to our documentation and others to Domo? Is it expected?

File: openmetadata-docs/content/connectors/pipeline/domo-pipeline/index.md (hunk 2 above)
Comment (NiharDoshi99) [reply openmetadata-docs/content/connectors/pipeline/domo-pipeline/index.md:45]: yes

File: openmetadata-docs/content/connectors/dashboard/domo-dashboard/cli.md (hunk 3 above)
Comment (NiharDoshi99) [reply openmetadata-docs/content/connectors/dashboard/domo-dashboard/cli.md:26]: yes 👍

**Review by nahuelverdugo — APPROVED** [review]

**Review by MilanBariya — APPROVED** [review]

**Pull request merged**